packages/rancher-webhook/package.yaml

```diff
@@ -1,2 +1,2 @@
-url: https://github.com/rancher/webhook/releases/download/v0.1.4-rc1/rancher-webhook-0.1.4-rc1.tgz
-packageVersion: 0
```

Comment:
Why was this removed?

Reply:
accident i would suspect. putting it back